pkg/test/istioio/command.go

```diff
@@ -232,71 +221,12 @@ func (s Command) getVerifier() Verifier {
 	return defaultVerifier
 }
 
-// TokenVerifier tokenizes the output and compares against the tokens from the given file.
-func TokenVerifier(selector InputSelector) Verifier {
-	return func(ctx Context, name, output string, _ error) {
-		input := selector.SelectInput(ctx)
-
-		// Read the input.
-		expected, err := input.ReadAll()
-		if err != nil {
-			ctx.Fatalf("verification failed for command %s: %v", name, err)
-		}
-
-		// Tokenize the content and the file.
-		expectedTokenLines := tokenize(expected)
-		actualTokenLines := tokenize(output)
-
-		if len(expectedTokenLines) != len(actualTokenLines) {
-			ctx.Fatalf("verification failed for command %s: line count: (expected %d, found %d). Expected:\n%s\nto match:\n%s",
-				name, len(expectedTokenLines), len(actualTokenLines), output, expected)
-		}
-
-		for lineIndex := 0; lineIndex < len(expectedTokenLines); lineIndex++ {
-			expectedTokens := expectedTokenLines[lineIndex]
-			actualTokens := actualTokenLines[lineIndex]
-
-			if len(expectedTokens) != len(actualTokens) {
-				ctx.Fatalf("verification failed for command %s [line %d]: token count (expected %d, found %d). Expected:\n%s\nto match:\n%s",
-					name, lineIndex, len(expectedTokens), len(actualTokens), output, expected)
-			}
-
-			for tokenIndex := 0; tokenIndex < len(expectedTokens); tokenIndex++ {
-				expectedToken := expectedTokens[tokenIndex]
-				if expectedToken == "?" {
-					// The value was a wildcard, matches anything.
-					continue
-				}
-
-				actualToken := actualTokens[tokenIndex]
-				if expectedToken != actualToken {
-					ctx.Fatalf("verification failed for command %s [line %d]: token %d (expected %s, found %s). Expected:\n%s\nto match:\n%s",
-						name, lineIndex, tokenIndex, expectedToken, actualToken, output, expected)
-				}
-			}
-		}
-	}
-}
-
-func tokenize(content string) [][]string {
-	inputLines := strings.Split(strings.TrimSpace(content), "\n")
-	tokenLines := make([][]string, 0)
-
-	for _, inputLine := range inputLines {
-		tokens := make([]string, 0)
-
-		var s scanner.Scanner
-
-		// Configure the token characters.
-		s.IsIdentRune = func(ch rune, i int) bool {
-			return ch == '_' || ch == '-' || ch == '.' || unicode.IsLetter(ch) || unicode.IsDigit(ch)
-		}
-
-		s.Init(strings.NewReader(inputLine))
-		for tok := s.Scan(); tok != scanner.EOF; tok = s.Scan() {
-			tokens = append(tokens, s.TokenText())
-		}
-		tokenLines = append(tokenLines, tokens)
+// SleepCommand returns a command that runs `sleep <delay>`
```

Comment:
Do we need a sleep command? This can be done in scripts and hopefully we can usually avoid sleep and watch for things appropriately.

Reply:
That's a short term fix while we port the logic found in `Sidecar` echo component to allow us to inspect the config of the proxy.

That said, docs usually say: Apply this config and wait a few seconds to get it propagated to the pods. This Sleep command mimics that behavior.